tortoise/recover.go

```diff
@@ -25,6 +25,18 @@ func Recover(db *datastore.CachedDB, beacon system.BeaconGetter, opts ...Opt) (*
 	if err != nil {
 		return nil, fmt.Errorf("failed to load latest known layer: %v", err)
 	}
+
+	if types.GetEffectiveGenesis() != types.FirstEffectiveGenesis() {
+		first := types.GetEffectiveGenesis().GetEpoch()
+		for _, epoch := range []types.EpochID{first, first + 1} {
+			if err := db.IterateEpochATXHeaders(epoch, func(header *types.ActivationTxHeader) bool {
+				trtl.OnAtx(header)
+				return true
+			}); err != nil {
+				return nil, err
+			}
+		}
+	}
```

Comment:
there is a function recoverEpoch now

i think the only thing that is missing is:


```
if ... {
      recoverEpoch(first)
}
```

first+1 will be recovered either if latest is 0, or normally in the loop